runtime/dancebox/src/lib.rs

```diff
@@ -36,6 +36,7 @@ pub mod weights;
 #[cfg(test)]
 mod tests;
 
+use pallet_collator_assignment::CoreAllocationConfiguration;
```

Comment:
same thing, put it inside the rest of the dependencies please

Reply:
*fixed*